setup.py

```diff
@@ -28,7 +28,7 @@
     package_dir={"": "src"},
     install_requires=[
         "amazon-braket-schemas>=1.18.0",
-        "amazon-braket-default-simulator>=1.19.0",
+        "amazon-braket-default-simulator @ git+https://github.com/aws/amazon-braket-default-simulator-python.git@task-executor",  # noqa
```

Comment:
Once you've merged the default sim changes, remember to change this branch
```suggestion
        "amazon-braket-default-simulator @ git+https://github.com/aws/amazon-braket-default-simulator-python.git@feature/execute-manager",  # noqa
```